pkg/kubectl/cmd/drain.go

```diff
@@ -585,6 +585,7 @@ func (o *DrainOptions) evictPods(pods []corev1.Pod, policyGroupVersion string, g
 					doneCh <- true
 					return
 				} else if apierrors.IsTooManyRequests(err) {
+					fmt.Fprintf(o.ErrOut, "error when evicting pod %q: %s will retry after 5s\n", pod.Name, err.Error())
```

Comment:
do you want to match the `%v` for err? like other lines in this file? (also `err` instead of `err.Error()`

Reply:
thanks @dims I've pushed the suggested change.